### PR TITLE
Add cluster fileld in QueueSpec to support multi-cluster-scheduling case

### PR DESCRIPTION
**Pull request by william-wang**
Signed-off-by: w00503381 <wangleibo1@huawei.com>

### CODE REVIEW

**Review by Thor-wl — COMMENTED** [review]
Is there some design docs for that?

**Comment by william-wang** [comment]
The design doc will be ready soon.

File: pkg/apis/scheduling/types.go [hunk 1]
```diff
@@ -278,6 +278,13 @@ type QueueStatus struct {
 	Reservation Reservation `json:"reservation,omitempty" protobuf:"bytes,6,opt,name=reservation"`
 }
 
+// CluterSpec represents the template of Cluster
+type Cluster struct {
+	Name string
+	Weight int32
```
Comment (Thor-wl) [inline pkg/apis/scheduling/types.go:284]: Is there some comments about field `Weight` and `Capacity`?

Comment (william-wang) [reply pkg/apis/scheduling/types.go:284]: The meaning of cluster Weight and Capacity is straightforward. It is similiar with Weight and Capacity in QueueSpec.

Comment (Thor-wl) [reply pkg/apis/scheduling/types.go:284]: > The meaning of cluster Weight and Capacity is straightforward. It is similiar with Weight and Capacity in QueueSpec.

OK, IC

**Review by Thor-wl — APPROVED** [review]

**Review by wpeng102 — APPROVED** [review]
/lgtm

**Review by wpeng102 — APPROVED** [review]
/approve

**Review by wpeng102 — APPROVED** [review]

**Review by Thor-wl — APPROVED** [review]

**Pull request closed without merging**